### PR TITLE
Move CI/CD from Travis to Github Actions

### PR DESCRIPTION
**Pull request by kylem038**
Goal of this PR is to use Github actions to control CI/CD and no longer use TravisCI. 

Currently the test suite will run on push to any of the branches on the repo. 
Only if the push is to `develop`, and only if the test step passes, will the deploy job run. 

Currently have 1 test assertion commented out which I will follow-up on later.

### CODE REVIEW

File: build/build.js [hunk 1]
```diff
@@ -33,4 +33,5 @@ webpack(webpackConfig, function (err, stats) {
     chunks: false,
     chunkModules: false
   }) + '\n')
+  process.exit(0);
```
Comment (kylem038) [inline build/build.js:36]: Needed webpack to kick out after it completed or else it just hangs during CD.

File: test/e2e/nightwatch.conf.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-require('babel-register')
```
Comment (kylem038) [inline test/e2e/nightwatch.conf.js:1]: Sorry - this is my formatter doing some fixing.

**Comment by bgschiller** [comment]
Cool, this looks good. You're wanting to fix the commented-out assertion in a follow-up PR?

Could you remove the travis-ci files in this same branch, so they're not both trying to run?

**Comment by kylem038** [comment]
@bgschiller yeah I was going to address it in a follow-up, but if we want to wait until tomorrow I could possibly get the fix into this PR. 

Good catch on the Travis files. I deleted them on another branch.

**Pull request merged**